packages/rest/src/middlewareTemplate.ts.hbs

```diff
@@ -0,0 +1,201 @@
+/* tslint:disable */
```

Comment:
Do we need to push this?

Reply:
Yes, we need a custom template because it doesn't call `next()` at the end by default, and we need to have "afterware" to be able to close a tenant session. 

Spent a lot of time on this, and this is the only way I could find, sadly